frontend/www/js/omegaup/components/homepage/CookieConsent.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <vue-cookie-accept-decline
+    ref="myPanel1"
+    :debug="false"
+    :disable-decline="false"
+    :show-postpone-button="false"
+    element-id="myPanel1"
+    position="bottom-left"
+    transition-name="slideFromBottom"
+    type="floating"
+    @clicked-accept="$emit('cookie-clicked-accept')"
+    @clicked-decline="$emit('cookie-clicked-decline')"
+    @clicked-postpone="$emit('cookie-clicked-postpone')"
+    @removed-cookie="cookieRemovedCookie"
+  >
+  </vue-cookie-accept-decline>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import VueCookieAcceptDecline from 'vue-cookie-accept-decline';
+import 'vue-cookie-accept-decline/dist/vue-cookie-accept-decline.css';
+@Component({
+  components: {
+    'vue-cookie-accept-decline': VueCookieAcceptDecline,
+  },
+})
+export default class Cookie extends Vue {
```

Comment:
The filename for this component changed, so I recommend you change the class name too:
```suggestion
export default class CookieConsent extends Vue {
T = T;
```
Also, you need to add `T = T;` inside the class in order to avoid this error:
```
frontend/www/js/omegaup/components/homepage/CookieConsent.vue:5:8: [eslint] 'T' is defined 
but never used. [@typescript-eslint/no-unused-vars] :disable-decline="false" ^ Errors cannot be 
automatically fixed. error: failed to push some refs to 'https://github.com/anmode/omegaup
```

Reply:
ohh okay! Done, Thanks